assets/src/edit-story/components/panels/document/publish/publishTime.js

```diff
@@ -117,6 +120,7 @@ function PublishTime() {
           ref={dateFieldRef}
           activeItemLabel={displayLabel}
           selectButtonStylesOverride={focusStyle}
+          onPointerEnter={() => loadCalendar()}
```

Comment:
Do we also want `onFocus={() => loadCalendar()}` here?

Reply:
Yep, good point!